proto/reaction.proto

```diff
@@ -209,6 +211,31 @@ message UnmeasuredAmount {
   string details = 2;
 }
 
+/**
+ * This qualitatively describes the apparent size and morphology of a Compound
+ * or a ReactionInput.
```

Comment:
include product in this list as well

Reply:
fixed in https://github.com/open-reaction-database/ord-schema/pull/698/commits/c03e02687ae6831084669afc9a01eee6e3984d55